src/guide/android_troubleshooting.md

```diff
@@ -0,0 +1,76 @@
+# Troubleshooting Android
+
+When including the Breez SDK and/or Notification Plugin into your application you might come across issues compiling because: 
+- it uses `kotlinx-serialization`
+- it relies on JNA library
```

Comment:
```suggestion
After installing Breez SDK into your application you might come across issues compiling on Android platforms because Breez SDK's Notification Plugin: 
- uses `kotlinx-serialization` dependency,
- and relies on JNA library.
```